app/views/grids/_notes.html.erb

```diff
@@ -78,7 +78,6 @@
 </table>
 </div>
 
-<a class="grid-embed grid-embed-<%= randomSeed %>" title="Embed this table on another site."><i class="fa fa-code" style="color:#aaa;"></i></a>
```

Comment:
Actually we would like to keep the embed codes for notes and wikis! If you can drop these removals from your PR this looks good to go, thank you!